Installation Guide

pyrevo opened the pull request:
Hi, we attempted to write an installation guide. I tried to follow it step by step and was able to configure Twist_DNA on a second workstation and perform a complete analysis with the latest version (commit: a4404ee9a9a96f1048c18435a94e01a1d7230380).

I also added just a few lines to the config file template (configdefaults201012.yaml) from the previous versions to make it work on the first attempt.

Many thanks,
Massi

Commit:
docu

Commit:
docu

Commit:
docu

pyrevo committed:
Added documentation

pyrevo committed:
Added documentation

Smeds commented:
Great! I will take a look at it

Smeds commented on README.md:
```diff
@@ -3,3 +3,187 @@
+## System configuration
+
+### Requirements
+The easier way is to create a conda environment by following these steps:
```
Looks good! I will later add a section of how to setup up the environment where you have access to singularity. All those dependencies aren't required when using our singularity containers.

pyrevo replied:
This sounds good, thank you! Please feel free to add, delete or modify whatever is needed and if there is any suggestion to improve it let me know and I will do it.

With Lauri, we are also testing Twist_DNA on Red Hat Enterprise Linux v.8.4 and we have found few different issues that we might have overcame. As soon as we get the pipeline work 100% on the clinical side he might update this readme with instructions for users experiencing these issues.

Then maybe we can try to see if we are able to help with some of the pending tasks issued on the development side.

ljmesi committed:
Add info on adjustments on certain cases

The adjustments are applicable for rule recall when using RHEL 8 and in
case one wishes to run coverage analysis with bedtools coverage.

pyrevo committed:
Merge pull request #1 from ljmesi/CGL-Documentation

Added issue with Red Hat Enterprise Linux 8.4

pyrevo committed:
Merge branch 'clinical-genomics-uppsala:develop' into CGL-Documentation
